tests/utils.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+
+def get_resource_path(filename):
```

Comment:
I've had issues with static resource (`json` files for schema definitions) when packaged. I would encourage the use of [`pkg_resources`](http://setuptools.readthedocs.io/en/latest/pkg_resources.html)